packages/integration-tests/index.js

```diff
@@ -94,25 +94,30 @@ async function getExecutionStatus(executionArn, retryOptions) {
 async function waitForCompletedExecution(executionArn, timeout = 600) {
   let executionStatus;
   let iteration = 0;
+  const sleepPeriodMs = 5000;
+  const maxIterationsToStartExecution = 24; // 2 minutes
+  const iterationsPerMinute = Math.floor(60000 / sleepPeriodMs);
   const stopTime = Date.now() + (timeout * 1000);
+
   /* eslint-disable no-await-in-loop */
   do {
     iteration += 1;
     try {
       executionStatus = await getExecutionStatus(executionArn);
     }
     catch (err) {
-      if (!(err.code === 'ExecutionDoesNotExist') || iteration > 12) {
+      if (!(err.code === 'ExecutionDoesNotExist') || iteration > maxIterationsToStartExecution) {
```

Comment:
@Jkovarik This current || will throw an exception in the case that getExecutionStatus is throttled.  Is that really what was intended?

Reply:
On further thought, since we're 5 secs between calls, I think this is irrelevant.